### DISCUSSION TITLE
How to run command from extension code and restart if this already running?

### DISCUSSION THREAD

**Discussion by SeryiBaran** [microsoft/vscode-discussions#1213] [Extension Development QnA] [answered]
I need to run command platformio-ide.build with vscode.commands.executeCommand, but if this command already running, VS Code asks The task 'PlatformIO: Build (ardustat)' is already active. [Terminate task] [Restart task]. I need to automatically restart the command in this case. What do I need to do? I've tried to read VS Code docs, but not found anything.

**Comment by isidorn** [comment, marked as answer]
I do not think this is possible. The command would have to support that as an argument.
